service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -540,25 +536,6 @@ fun Database.Read.getCitizenChildren(citizenId: PersonId): List<CitizenChildren>
         .list()
 }
 
-private val toPersonApplicationSummary: (ResultSet, StatementContext) -> PersonApplicationSummary = { rs, _ ->
```

Comment:
Several column names are different, so I don't think you can remove this manual mapping without modifying the queries a bit

Reply:
Good catch! Luckily this was also caught by tests. Fixed.